vaadin-charts-flow-parent/vaadin-charts-flow-svg-generator/jsdom-exporter.js

```diff
@@ -191,14 +193,24 @@ const inflateFunctions = (jsonConfiguration) => {
 
 /**
  * Function to export SVG a string containing a chart based
- * on the configuration provided
+ * on the configuration provided.
+ *
+ * - The `chartConfiguration` property can be used to provide a configuration when the exporter is being called from another JS script.
+ * - The `chartConfigurationFile` property can be used when the exporter is being called via CLI to get around the max argument length limit.
+ *
+ * The `chartConfigurationFile` property takes priority over `chartConfiguration`.
  *
  * @param {ExportConfiguration} configuration
  *
  * @returns {Promise<SVGResult>} Object with the result of the export
  */
-const jsdomExporter = ({ chartConfiguration, outFile = 'chart.svg', exportOptions }) => {
+const jsdomExporter = ({ chartConfigurationFile, chartConfiguration, outFile = 'chart.svg', exportOptions }) => {
     return new Promise((resolve, reject) => {
+        if (chartConfigurationFile) {
+            chartConfiguration = JSON.parse(
+                fs.readFileSync(path.join(__dirname, chartConfigurationFile)).toString()
```

Comment:
```suggestion
                fs.readFileSync(path.join(__dirname, chartConfigurationFile), 'utf8').toString()
```
One concern about `fs.readFileSync` is the text encoding to be used for converting the byte data into a string. I can't see any clear information in the node.js docs which encoding it will use if you don't explicitely specify one, I guess it might be the system default. This can lead to broken characters if you provide a UTF-8 encoded file, and read it on a system that uses a different encoding. Since we [explicitely save the file using UTF-8 encoding](https://github.com/vaadin/flow-components/pull/2836/files?diff=unified&w=0#diff-ba2c17cca75426fd099e91330f1467becd8c210d950b189d040786ed5193646cR174-R175), I think we can also explicitely read it as UTF-8.

Reply:
Good point, applied.